src/roots.jl

```diff
@@ -119,100 +54,162 @@ IntervalLike{T} = Union{Interval{T}, IntervalBox{T}}
 NewtonLike = Union{Type{Newton}, Type{Krawczyk}}
 
 """
-    roots(f, X, contractor, tol=1e-3)
-    roots(f, deriv, X, contractor, tol=1e-3)
+    roots(f, X, contractor=Newton, strategy=BreadthFirstSearch, tol=1e-15)
```

Comment:
In line L80 the same value is defined as a constant. It might be worth moving the constant up to reuse it here (and on the following line).